contracts/crc/CRCV0_2_0.sol

```diff
@@ -0,0 +1,24 @@
+pragma solidity ^0.4.24;
+import "./CRCV0_1_0.sol";
+
+
+contract CRCV0_2_0 is CRCV0_1_0 {
+
+  /// @notice Returns the total value of crcs owned by a specific address.
+  /// @param _owner The owner address to check.
+  function balanceOf(address _owner) public view returns (uint256 totalValue) {
+    uint256 bundleBalance = BasicCommodity.balanceOf(_owner);
+    uint256 supply = BasicCommodity.totalSupply();
+    uint totalCrcValue = 0;
+    
+    for (uint i = 0; i < supply || i < bundleBalance; i++) {
+      address ownerToCheck = BasicCommodity.ownerOf(i);
+      if (ownerToCheck == _owner) {
+        totalCrcValue += BasicCommodity.getCommodityValueByIndex(i);
+      }
+    }
+    
+    return totalCrcValue;
```

Comment:
I was imagining we would denormalize the data and store a separate balance int somewhere. I guess the relevant questions to ask are:

- will we ever call balanceOf from inside a contract, in which case this function could end up costing a lot of gas
- how many bundles do we expect people to have?
- how error prone is it to keep a denormalized sum field in sync?

What are your thoughts here?

Reply:
maybe using `mBalnces`? (currently unused). I dont imagine we ever use this inside a contract. Previously we had a JS function that did this same thing external of the blockchain, but that was removed when we deprecated the old `contracts.js` file

Reply:
I guess I would vote for using something like mBalances.

Reply:
Ok. trying to find a delicate balance between "getting features done" and "implementing features right using upgrades". Thoughts on handling this by updating the base files vs handling it via a new version?

Reply:
it requires changes to `mint` , and if we use the upgrade-only journey, we are going to end up with some pretty unreadable code. I think we should reserve upgrades for live changes, and should probably keep to just modifying base files as much as possible until we get the closest thing to a working base first. Thoughts?

Reply:
basically-  its entirely possible to preserve state and do this in an upgraded version, but im not sure theres any state worth preserving just yet. The tradeoff is readability